src/devices/Si7021/README.md

```diff
@@ -1,21 +1,29 @@
-﻿# Using Si7021 (temperature and humidity sensor)
-
-## Summary
-
+﻿# Si7021
```

Comment:
Perhaps `SI7021 - temperature and humidity` good call on removing stuff like `using` etc here